bazel/python_test.bzl

```diff
@@ -92,7 +92,7 @@ _pytest_runner = rule(
     },
 )
 
-def pytest_test(name, srcs, deps = [], args = [], data = [], python_version = None, **kwargs):
+def pytest_test(name, srcs, deps = [], args = [], data = [], imports = [], python_version = None, **kwargs):
```

Comment:
Have you considered maybe implicitly importing both Lte and Orc8r paths for tests? Or maybe defining a `magma_pytest_test` wrapper to do all the Magma specific defaults.

Reply:
(This is out of scope for this PR, so feel free to not take action on this here though :D ) I just wanted to get your opinion on that.

Reply:
I would like to do that ... but, paths in imports are relative (because the sandbox root path changes all the time, e.g. ´/tmp/bazel/sandbox/23/foo´) and the actual relative path depends on the folder where the package of the target resides, e.g., the LTE_ROOT can be `../../` or `../../../` ... . If you have an idea/hint how to approach this, then let me know.